workspaces/libnpmpublish/lib/provenance.js

```diff
@@ -63,7 +82,142 @@ const generateProvenance = async (subject, opts) => {
     },
   }
 
-  return sigstore.attest(Buffer.from(JSON.stringify(payload)), INTOTO_PAYLOAD_TYPE, opts)
+  return Buffer.from(JSON.stringify(payload))
```

Comment:
the buffer and json parts seem to apply to both, should they be hoisted out to the caller too?

Reply:
Done.